<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MQTTForm</class>
 <widget class="QWidget" name="MQTTForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="4,3">
       <item>
        <widget class="QTextBrowser" name="LogBrowser"/>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="ServerIP">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>服务器地址：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ServerIPLine">
               <property name="text">
                <string>iot-06z00hbczg65rf0.mqtt.iothub.aliyuncs.com</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ServerPortLine">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>1883</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="ClientID">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>客户端ID：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ClientIDLine">
               <property name="text">
                <string>Test|securemode=2,signmethod=hmacsha1,timestamp=1714204000504|</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="User">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>用户名:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="UserLine">
               <property name="text">
                <string>Node1&amp;k14suPxlVyQ</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="Password">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>密码：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="PasswordLine">
               <property name="text">
                <string>0EF2B097FC2D6856A171BED08C6EB4B1CBFF4A45</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="ConnectBtn">
             <property name="text">
              <string>连接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Ping</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="3">
              <widget class="QSpinBox" name="QosBox"/>
             </item>
             <item row="1" column="4">
              <widget class="QPushButton" name="PublishBtn">
               <property name="text">
                <string>发布</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="Topic">
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>订阅主题：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="SubTopicLine"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="Message">
               <property name="minimumSize">
                <size>
                 <width>45</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>发布主题：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="MesTopicLine">
               <property name="text">
                <string>/sys/k14suPxlVyQ/Node1/thing/service/property/set</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="UnSubscribeBtn">
               <property name="text">
                <string>取消订阅</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QPushButton" name="SubscribeBtn">
               <property name="text">
                <string>订阅</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="MessageEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
